lib/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as promClient from "prom-client";
+import * as express from "express";
+
+// TODO: this shoudl be all of promClient
+// not sure how to get that to work in a declaration file
+export const client: any; 
```

Comment:
Can you use `typeof promClient`?

Reply:
My attempt:
```
import * as client from "prom-client";
export { client };
```

Reply:
I've got it linked to another project and typeof seems to work. Feels a bit strange but;
https://blog.mariusschulz.com/2016/05/31/type-queries-and-typeof-in-typescript#typescripts-type-queries

`export const client: typeof promClient;`